test/TEST-21-SYSUSERS/test-2.input

```diff
@@ -1,4 +1,7 @@
 # Trivial smoke test that generate the ID dynamically based on SYSTEM_UID_MAX
```

Comment:
A bit of a nitpick, but the test description does no longer match the test. I would prefer to either update the description or move the test into e.g. test-N.input with an appropriate description.

Reply:
Updated.